packages/components/src/components/animations/recolorLottieAnimation.tsx

```
@@ -0,0 +1,78 @@
+                    if (replacement) {
+                        node.splice(i, 4, ...replacement);
+                        i += 3;
+                    }
+                }
+            }
+
+            node.forEach(walk);
+        } else if (typeof node === 'object' && node !== null) {
+            for (const key in node) {
+                // eslint-disable-next-line no-prototype-builtins
+                if (node.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)